src/appleseed.shaders/src/appleseed/as_standard_surface.osl

```diff
@@ -672,7 +686,7 @@ shader as_standard_surface
 
         if (in_fresnel_type == 0)
         {
-            if (compute_transmission)
+            if (!compute_coating && compute_transmission)
```

Comment:
Isn't that a regression? @est77 removed this check that was causing things to become black when coating was applied, IIRC.

Reply:
When this shader was done, coating on glass would generate a huge amount of noise hence it was disabled since it was unusable. This was kind of suspended pending work on a mechanism to layer BxDFs, we then had as_layer() closure. I haven't revisited this particular case yet, but if @est77 removed this condition then i assume the coating on BTDF case is already correct.

Reply:
We did some testing after removing the condition and results looked correct.
Do you have any particular scene where the result was incorrect?